src/VisualStudio/Core/Test.Next/Services/LspDiagnosticsTests.cs

```diff
@@ -364,12 +364,6 @@ public async Task ClearAllDiagnosticsForMappedFileToManyDocumentsTestAsync()
             jsonRpc.SynchronizationContext = new RpcOrderPreservingSynchronizationContext();
             jsonRpc.StartListening();
 
-            // Triggers language server to send notifications.
-            foreach (var document in documentsToPublish)
-            {
-                await languageServer.PublishDiagnosticsAsync(document).ConfigureAwait(false);
```

Comment:
Are these tests actually testing anything any more, if the publish diagnostics has been removed? Or is the file name misleading?

Reply:
These are actually removed now.  Replaced with new pull-diagnostic tests.